docs/Project-Configuration/Project-Commands.md

```diff
@@ -768,6 +768,188 @@ Parameters:
+| `prefix`                  | string          | The S3 bucket's prefix (if any).                                               |
+| `permissions`             | string          | The S3 canned ACL permission to apply to the uploaded artifact.                |
+| `path`                    | string          | The artifact's unique S3 object key (usually the metric name / FTDC filename). |
+| `tags`                    | list of strings | The artifact's list of tags.                                                   |
+| `created_at`              | timestamp       | The artifact's creation time.                                                  |
+| `is_text`                 | boolean         | (Optional) The artifact is a plain text file.                                  |
+| `is_ftdc`                 | boolean         | (Optional) The artifact is an FTDC file.                                       |
+| `is_bson`                 | boolean         | (Optional) The artifact is a BSON file.                                        |
+| `is_json`                 | boolean         | (Optional) The artifact is a JSON file.                                        |
+| `is_csv`                  | boolean         | (Optional) The artifact is a CSV file.                                         |
+| `is_uncompressed`         | boolean         | (Optional) The artifact is an uncompressed file.                               |
+| `is_gzip`                 | boolean         | (Optional) The artifact is a GZIP file.                                        |
+| `is_tarball`              | boolean         | (Optional) The artifact is a tarball.                                          |
+| `convert_gzip`            | boolean         | (Optional) Should gzip the artifact before uploading.                          |
+| `convert_bson_to_ftdc`    | boolean         | (Optional) Should convert the BSON artifact file to FTDC before uploading.     |
+| `convert_json_to_ftdc`    | boolean         | (Optional) Should convert the JSON artifact file to FTDC before uploading.     |
+| `convert_csv_to_ftdc`     | boolean         | (Optional) Should convert the CSV artifact file to FTDC before uploading.      |
+| `events_raw`              | boolean         | (Unused)                                                                       |
+| `events_histogram`        | boolean         | (Unused)                                                                       |
+| `events_interval_summary` | boolean         | (Unused)                                                                       |
+| `events_collapsed`        | boolean         | (Unused)                                                                       |
+
+**Note:** Here is the list of [valid S3 canned ACL permissions](https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html#canned-acl).
```

Comment:
I would add this to the `permissions` field description, something like: 
```
The S3 canned ACL permission to apply to the uploaded artifact. See the list of valid permissions [here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html#canned-acl).
```

Reply:
Done. Thanks!